docs/_docs/collections.md

```diff
@@ -51,7 +51,7 @@ defaults:
 
 Create a corresponding folder (for example, `<source>/_my_collection`) and add
 documents. YAML Front Matter is read in as data if it exists, and everything
-after it is available in the document's `content` attribute. If no YAML Front
+after it is accessible via the document's `content` attribute. If no YAML Front
```

Comment:
I don't want to continue arguing over this.. just one last attempt..
Read through the following:

`"Everything after it is available online"` &mdash; this denotes a point-in-time after *something is available online*. (the noun is just `Everything`)
`"Everything after it, is available online"` &mdash; this denotes a point-in-time right now and beyond. (the noun is `Everything after it` referring to whatever there is, after the object `it`. (it == FrontMatter dashes in our original context.)

Upvote if I'm in the right, else downvote. :smiley: